### DISCUSSION TITLE
Overriding which-key "extra" window keymaps

### DISCUSSION THREAD

**Discussion by nianiam** [LazyVim/LazyVim#4129] [Q&A] [answered]
The new Which key v3's expand extra loads a bunch of built in window keybinds into which key. This is now the default in LazyVim:

```lua
{
  "<leader>w",
  group = "windows",
  proxy = "<c-w>",
  expand = function()
    return require("which-key.extras").expand.win()
  end,
},
```

For the most part this is very useful, but I have a few keymaps which are getting overidden. Some of my keymaps are defined in `lua/config/keymaps.lua` and some in the `keys` option for the plugin I am using. An example is `<leader>wd` which the whichkey expansion overrides to `Delete Window`. Is there a way of modifying this expansion to ignore certain keybinds from `require("which-key.extras").expand.win()` or can I define my own keymaps in a way that takes preference over this?

**Comment by dpetka2001** [comment, marked as answer]
If you set your own keymap it should overwrite the default. Make sure `which-key` is updated to latest version. This was added a few days ago.

**Reply by nianiam** [reply]
Thanks, updated and it's working.
